### ISSUE TITLE
Set up waffle board

### ISSUE THREAD

**Issue by acka47** [hbz/oerworldmap#596]
@literarymachine and me think that it make sense to use a waffle board instead of huboard. It is more useful and also used by other hbz projects. I will add the needed labels and set up the board.

**Comment by acka47** [comment]
I am not sure what we said about the "Tasklog" column. We don't have this for lobid projects and I think we don't need it for OER Worldmap. Should I leave it out for the waffle board?

**Comment by literarymachine** [comment]
> I am not sure what we said about the "Tasklog" column.

As far as I remember, we agreed to remove this column and add a dedicated one for user stories instead.

**Comment by acka47** [comment]
I configured the waffle board at https://waffle.io/hbz/oerworldmap. The huboard board doesn't work anymore. As discussed above I removed the "Tasklog" column and added a "stories" column. As Stories don't go through the columns from left to right, strange things happen when a story is closed (it lands in a column in between "stories" and "Done". That's why a story has to be archived as soon as it is closed.

I think the rest is working fine. I will take some time going through the issues, closing the ones that are fixed and doing some ordering.
